a3p-integration/proposals/z:acceptance/test-lib/kread.js

```diff
@@ -1,4 +1,5 @@
 // @ts-nocheck FIXME
+// XXX uses agoric.follow to read data through spawned processes; replace with VstorageKit
```

Comment:
Question: is agoric.follow deprecated? Just not preferred? Is there a larger pattern of work to replace it with vstorage kit that we should file?

Reply:
Deprecated, yeah. The larger pattern is in the comment, "replace with VstorageKit". That gets the vstorage data without shelling out, making it more portable and less dependent on the process env (e.g. PATH)